src/app/innstillinger/modal/innstillinger-modal-form.tsx

```diff
@@ -45,6 +47,15 @@ const AutomatiskeTagsLabel = (
         </Hjelpetekst>
     </Label>
 );
+const OppgaveDestinasjonLabel = (
+    <Label>
+        <b>Oppgave destinasjon ved svar</b>
```

Comment:
Usikker på om det egentlig er "oppgavedestinasjon" i ett ord? Ref "reisedestinasjon".
Evt så er det kanskje bedre å skrive noe à la "Destinasjon for oppgave ved svar"?